Dizzy/Screens/Discovery/Cells/DiscoveryPlaceCell.swift

```diff
@@ -23,7 +23,7 @@ class DiscoveryPlaceCell: UICollectionViewCell {
     
     let stackViewTrailingPadding = CGFloat(15)
```

Comment:
on real device, the press on place text is not good because other place gets the press

Reply:
Works as charm!